js/base/Exchange.js

```diff
@@ -2655,4 +2655,16 @@ module.exports = class Exchange {
         }
         return undefined;
     }
+
+    handleMarginMode (params = {}) {
+        /**
+         * @ignore
+         * @method
+         * @param {dict} params extra parameters specific to the exchange api endpoint
+         * @returns {str|undefined} the marginMode in lowercase as specified by params["marginMode"], this.options["marginMode"] or this.options["defaultMarginMode"]
+         */
+        const defaultMarginMode = this.safeStringLower2 (this.options, 'marginMode', 'defaultMarginMode');
+        const marginMode = this.safeStringLower (params, 'marginMode', defaultMarginMode);
+        return marginMode;
```

Comment:
Same comment as in the other PR, probably we have to return params here with 'marginMode' omitted, similar to what we do in `handleMarketTypeAndParams`

Reply:
it's omitted in the `fetchBalance` method

Reply:
@samgermain anyone who calls that method has to remember that `marginMode` needs to be omitted. I think that might be a bit error-prone but If you think is better that way no problem